frodo.js

```diff
@@ -7,126 +7,122 @@
  * See README.md for details.
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.